tracing-fmt/src/writer.rs

```diff
@@ -0,0 +1,165 @@
+//! Abstractions for creating [`io::Write`] instances.
+
+use super::Builder;
+use std::io;
+
+/// A type that can create [`io::Write`] instances.
+///
+/// `NewWriter` is used by [`FmtSubscriber`] to print formatted text representations of [`Event`]s.
+///
+/// This trait is already implemented for function pointers and immutably-borrowing closures that
+/// return an instance of [`io::Write`], such as [`io::stdout`] and [`io::stderr`].
+///
+/// [`FmtSubscriber`]: crate::FmtSubscriber
```

Comment:
All the intra-RustDoc links in this file are 404s for me when I run
```
$ cargo doc --open -p tracing-fmt
```
and browse around. 

Elsewhere in the project we use relative URLs rather than intra-rustdoc links, since I think that style of linking still has some [issues that need to be worked out](https://github.com/rust-lang/rust/issues/43466). Can we use relative URLs here instead so that these links aren't broken?